### PR TITLE
CI: Convert Python2 testing scripts to Python3

### PR DESCRIPTION
**Pull request by Alexey-Rivkin**
## What?
Convert Python2 testing scripts to Python3.

## Why?
We missed the Python2 EOL date a bit (Jan 1, 2020).

### CODE REVIEW

File: test/apps/test_fuzzy_match.py
Comment (dpressle) [inline test/apps/test_fuzzy_match.py:1]: Do we want to also fix the shebang to python3 and not be dependent on default python settings?

**Review by dpressle — CHANGES_REQUESTED** [review]

File: test/apps/test_ucx_tls.py
Comment (dpressle) [inline test/apps/test_ucx_tls.py:1]: python3?

**Review by dpressle — CHANGES_REQUESTED** [review]

File: test/apps/test_ucx_tls.py
Comment (Alexey-Rivkin) [reply test/apps/test_ucx_tls.py:1]: Sure, thanks!

File: test/apps/test_ucx_tls.py [hunk 1]
```diff
@@ -115,12 +114,12 @@ def _override_env(var_name, value):
...
     if options.verbose:
-        print cmd
+        print(cmd)
 
-    status, output = commands.getstatusoutput(cmd)
+    status, output = subprocess.getstatusoutput(cmd)
     if options.verbose:
-        print "return code " + str(status)
-        print output
+        print("return code " + str(status))
```
[the remark this answers is not in the document]
Comment (Alexey-Rivkin) [reply test/apps/test_ucx_tls.py]: Done

**Review by dpressle — APPROVED** [review]

**Review by yosefe — APPROVED** [review]

**Comment by yosefe** [comment]
@Alexey-Rivkin can u pls port to v1.18.x?

**Pull request merged**